src/app/core.clj

```diff
@@ -73,15 +66,7 @@
       (name2operation (:query opts))
       (nil? (:errors opts)))
     (run opts)
-    ;; The following code block is very similar to the usageError function.
-    ;; Try to call that method to avoid repeating code.
-    (do
-      (println "Incorrect usage")
-      (when (:errors opts)
-        (doseq [e (:errors opts)]
-          (println " * " e)))
-      (println banner)
-      (System/exit 1))))
+    (usageError banner opts "")))
```

Comment:
before we were calling System/exit 1, should we call it explicitly after usageError?